assets/style.css

```diff
@@ -0,0 +1,130 @@
+/*
+File: style.css
```

Comment:
For this file, maybe add a comment above every class just to tell the reader where this class is used? E.g. above .landing, '# used for the entire landing page', and above .s, where it is used/what it is used for. Just a simple small comment would do